### ISSUE TITLE
Is there a way to create a list of user accounts

### ISSUE THREAD

**Issue by mathieubossaert** [3liz/lizmap-web-client#836]
Good evening,

I implement Lizmap to help wine growers to proceed to self-diagnostics of their practices.
I use "filter by user" functionality and I need to create an account for each wine grower.
Is there a way or a command I can use to create users from a list of emails.

Thanks a lot,

mathieu

**Comment by mdouchin** [comment]
There is no Lizmap command yet to do so (import a list of users). You can have a look at jelix documentation: https://docs.jelix.org/fr/manuel/droits2/configuration#gerer-les-utilisateurs-dans-les-groupes

@laurentj do you know if such an import command if feasible with the new jcommunity module ?

**Comment by mathieubossaert** [comment]
Thnak you @mdouchin !
I will have a look at this. It already seems helpfull.

**Comment by laurentj** [comment]
warning, the documentation pointed by @mdouchin is to give rights to user, not to create accounts.

@mdouchin since there is no existing tools into jelix to import many users, everything is possible :) A script should be written.

**Comment by Gustry** [comment]
I agree, it would be nice to have an API to create users in bulk. (or SQL command)

**Comment by Gustry** [comment]
See https://github.com/3liz/lizmap-pgrouting-module/blob/master/tests/Makefile#L102 about the step `import-lizmap-acl`, getting close
